conduct.md

```diff
@@ -91,13 +99,13 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
+- Always be transparent on what are we doing wrong with. The reason being is because to create a mutual relationship environment between the team to understand the good and the bad of the activity together.
 - TODO
 - TODO
 
 #### Unacceptable Conduct
 
-- TODO
+- Never create unnecessary mistakes (Forget to git pull, never creating pull requests). This is very important to not create more problems and not to halt on the team's progress on the work.
```

Comment:
Maybe change 'This is very important to not create more problems' to something like 'It is very important not to create more problems'.